src/components/templates/Camp/hooks/usePartygoersOverlay.tsx

```diff
@@ -35,37 +32,33 @@ export const usePartygoersOverlay: ReactHook<
   partygoers,
   setSelectedUserProfile,
 }) => {
-  return useMemo(
-    () =>
-      // @debt this can be undefined because our types are broken so check explicitly
-      showGrid ? (
-        partygoers?.map(
-          (partygoer) =>
-            partygoer?.id && ( // @debt workaround, sometimes partygoers are duplicated but the new ones don't have id's
-              <MapPartygoerOverlay
-                key={partygoer.id}
-                partygoer={partygoer}
-                venueId={venueId}
-                myUserUid={userUid ?? ""} // @debt fix this to be less hacky
-                totalRows={rows}
-                totalColumns={columns}
-                withMiniAvatars={withMiniAvatars}
-                setSelectedUserProfile={setSelectedUserProfile}
-              />
-            )
-        )
-      ) : (
-        <div />
-      ),
```

Comment:
I believe this `<div />` was required here for some kind of styling related reasons. Have you changed those/checked that removing it doesn't break anything?